# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmStubs.cpp
# Commit: cd503f8f9a72
# Full Hash: cd503f8f9a72b8c701a587794f71405b36bcc809
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2020-10-20 14:55:47
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Part 7: Remove Frame::tls. r=lth
#   
#   We remove Frame::tls and fix prologue/epilogue. Runtime, Ion and Baseline are ready for this move already.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D83064
# ==============================================================================

diff -r 270961083337 -r cd503f8f9a72 js/src/wasm/WasmStubs.cpp
--- a/js/src/wasm/WasmStubs.cpp	Tue Oct 20 10:24:47 2020 +0000
+++ b/js/src/wasm/WasmStubs.cpp	Mon Oct 12 08:08:41 2020 +0000
@@ -2461,7 +2461,7 @@
     MIRTypeVector coerceArgTypes;
     MOZ_ALWAYS_TRUE(coerceArgTypes.append(MIRType::Pointer));
     unsigned offsetToCoerceArgv =
-        AlignBytes(StackArgBytesForWasmABI(coerceArgTypes), sizeof(Value));
+        AlignBytes(StackArgBytesForNativeABI(coerceArgTypes), sizeof(Value));
     MOZ_ASSERT(nativeFramePushed >= offsetToCoerceArgv + sizeof(Value));
     AssertStackAlignment(masm, ABIStackAlignment);
 